Dockerfile.dev

```diff
@@ -0,0 +1,16 @@
+FROM node:10
+
+RUN mkdir -p /usr/src
+WORKDIR /usr/src/
+
+COPY ./ .
+RUN chown -R node:node .
+
+USER node
+
+RUN yarn bootstrap
+
+EXPOSE 3000 6006 8080
+ENTRYPOINT ["yarn"]
+CMD ["workspace",  "@zooniverse/fe-project", "dev"]
```

Comment:
You can specify these last three lines in `docker-compose.yml` so it might be better to do that and avoid having a second Dockerfile.

Reply:
That sounds useful in terms of having a different entry point for each package too. Is the syntax the same in the YAML file?

Reply:
It passes the whole command as a string rather than a list, but here's an example: https://github.com/zooniverse/hamlet/blob/master/docker-compose.yml#L37

Reply:
Does that allow you to override the default args like ENTRYPOINT and CMD do? The string versions of those don't allow that in a Dockerfile.

Reply:
Yup I think they do.

Reply:
Docs say lists are supported so I might use those anyway.
https://docs.docker.com/compose/compose-file/#command
https://docs.docker.com/compose/compose-file/#entrypoint

All the documentation I read yesterday said to prefer arrays but didn't really explain why.